core/cas-server-core-util/src/main/java/org/apereo/cas/util/RegexUtils.java

```diff
@@ -15,7 +11,8 @@
  * @since 5.0.0
  */
 public final class RegexUtils {
-    private static final Logger LOGGER = LoggerFactory.getLogger(RegexUtils.class);
+
+    private static final Pattern MATCH_NOTHING_PATTERN = Pattern.compile("a^");
```

Comment:
Really? Please let me know which regex matches it. From my understanding, it means 'a' followed by the beginning of a string, which is already started, so it's not possible to match it.